Libraries/LibDesktop/Launcher.h

```diff
@@ -32,7 +32,7 @@ namespace Desktop {
 
 class Launcher {
 public:
-    static bool open(const URL&);
+    static bool open(const URL&, const Optional<String> handler_name={});
```

Comment:
`String` is already implicitly optional, as it has the null value (the wonders of C++), so why not just use that?

Reply:
Isn't C++ delightful! 🤮